packages/cli/src/commands/codegen.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable prefer-const */
+import { fixParameters } from "../lib/helpers/parameters";
+import { CodeGenerator } from "../lib/generators/CodeGenerator";
+
+import chalk from "chalk";
+import axios from "axios";
+import { GluegunToolbox } from "gluegun";
+
+const defaultTemplate = "web3api.gen";
+const defaultManifest = "web3api.yaml";
+
+const HELP = `
+${chalk.bold("w3 codegen")} ${chalk.bold("[<template-file>]")} [options]
+
+Template file:
+  Path to input template file (default: ${defaultTemplate})
+
+Options:
+  -h, --help                              Show usage information
+  -m, --manifest-path <path>              Path to the Web3API manifest file (default: ${defaultManifest})
+  -i, --ipfs [<node>]                     IPFS node to load external schemas (default: dev-server's node)
+  -o, --output-dir <path>                 Output directory for generated types (default: types/)
+  -e, --ens [<address>]                   ENS address
```

Comment:
Let's have a better description here, maybe: "ENS address to lookup external schemas (default: 0x0000...2e1e)